src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/admin/AdminPage.java

```diff
@@ -0,0 +1,41 @@
+package ca.corefacility.bioinformatics.irida.ria.integration.pages.admin;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import ca.corefacility.bioinformatics.irida.ria.integration.pages.AbstractPage;
+
+public class AdminPage extends AbstractPage {
+	public static final String RELATIVE_URL = "admin/";
+
+	@FindBy(tagName="body")
+	private WebElement body;
```

Comment:
Better just to find exactly what you are looking for here.  In this case you could find the title here.